### ISSUE TITLE
PopupButton keyboard navigation does not work

### ISSUE THREAD

**Issue by jreznot** [cuba-platform/cuba#1961]
### Environment

- Platform version: 7.0+
- Client type: Web
- Browser: Chrome, Firefox

### Description of the bug or enhancement

- Minimal reproducible example

1. Open Administration - Users
2. Click on Additional
3. Hit arrow down key

- Actual behavior

 Selected option is not changed.

See also Table context menu.
